### ISSUE TITLE
Can not find the 'Actions' button any more when update theme

### ISSUE THREAD

**Issue by GotzeWong** [abpframework/abp#9431]
Hi, 
   It's so glad to see the updates. But I met some new issues when I updated the themes with  https://github.com/abpframework/abp/pull/9208. 
  
### Issues

* ABP Framework version:  latest version.
* MVC  with Basic theme

* Steps needed to **reproduce** the problem.

 Run  "BackendAdminApp" Demo and UI is shown below at the first sight.

![image](https://user-images.githubusercontent.com/7664030/123359081-30ca1b00-d59f-11eb-8a12-7a87ae0690fa.png)

but the next second it became this 

![image](https://user-images.githubusercontent.com/7664030/123359193-6c64e500-d59f-11eb-9052-e7019bed123e.png)

* Exception message and **stack trace** if available (check the logs).
I also found this from the debugger of chrome.
![image](https://user-images.githubusercontent.com/7664030/123360565-1cd2e900-d5a0-11eb-8aea-d6a10d871a3b.png)

@maliming
I failed to find the answer from the official document about this updates.
Did I miss any steps? Please let me know and really appreciated it.

**Comment by realLiangshiwei** [comment]
Can you provide a sample project to reproduce?

**Comment by GotzeWong** [comment]
Hi, @realLiangshiwei，it's `AuthServer.Host` project from microservices demo with identity web module. 

I also found the related error from 'network' tab of chrome bebugger. 
![image](https://user-images.githubusercontent.com/7664030/123402556-70fabf00-d5da-11eb-8c30-7ff13abf29cf.png)

**Comment by realLiangshiwei** [comment]
You should check the application logs

**Comment by GotzeWong** [comment]
Followed @realLiangshiwei suggestion and checking the AuditLogging below


```
An exception occurred in the database while saving changes for context type 'Volo.Abp.AuditLogging.EntityFrameworkCore.AbpAuditLoggingDbContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> MySqlConnector.MySqlException (0x80004005): Data too long for column 'Exceptions' at row 1
```

And the exception is below

- Actions:
  Volo.Abp.AspNetCore.Mvc.ApplicationConfigurations.AbpApplicationConfigurationAppService.GetAsync (1198 ms.)
- Exceptions:
   An exception was thrown while activating Volo.Abp.Features.DefaultValueFeatureValueProvider -> Volo.Abp.FeatureManagement.FeatureStore -> Volo.Abp.FeatureManagement.FeatureManagementStore.
    Autofac.Core.DependencyResolutionException: An exception was thrown while activating Volo.Abp.Features.DefaultValueFeatureValueProvider -> Volo.Abp.FeatureManagement.FeatureStore -> Volo.Abp.FeatureManagement.FeatureManagementStore.
 ---> Autofac.Core.DependencyResolutionException: None of the constructors found with 'Volo.Abp.Autofac.AbpAutofacConstructorFinder' on type 'Volo.Abp.FeatureManagement.FeatureManagementStore' can be invoked with the available services and parameters:

Cannot resolve parameter 'Volo.Abp.FeatureManagement.IFeatureValueRepository featureValueRepository' of constructor 'Void .ctor(Volo.Abp.FeatureManagement.IFeatureValueRepository, Volo.Abp.Guids.IGuidGenerator, Volo.Abp.Caching.IDistributedCache`1[Volo.Abp.FeatureManagement.FeatureValueCacheItem], Volo.Abp.Features.IFeatureDefinitionManager)'.
  
 at Autofac.Core.Activators.Reflection.ReflectionActivator.GetAllBindings(ConstructorBinder[] availableConstructors, IComponentContext context, IEnumerable`1 parameters)
   at Autofac.Core.Activators.Reflection.ReflectionActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   at Autofac.Core.Activators.Reflection.ReflectionActivator.<ConfigurePipeline>b__11_0(ResolveRequestContext ctxt, Action`1 next)
   at Autofac.Core.Resolving.Middleware.DisposalTrackingMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Builder.RegistrationBuilder`3.<>c__DisplayClass41_0.<PropertiesAutowired>b__0(ResolveRequestContext ctxt, Action`1 next)
   at Autofac.Extras.DynamicProxy.RegistrationExtensions.<>c__DisplayClass8_0`3.<EnableInterfaceInterceptors>b__1(ResolveRequestContext ctxt, Action`1 next)
   at Autofac.Core.Resolving.Middleware.ActivatorErrorHandlingMiddleware.Execute(ResolveRequestContext context, Action`1 next)


Still working on this issue.

**Comment by GotzeWong** [comment]
Followed the exception and I found the exception occurs in `AbpApplicationConfigurationAppService.cs` line 89.

```
var result = new ApplicationConfigurationDto
            {
                Auth = await GetAuthConfigAsync(),
 line 89    //Features = await GetFeaturesConfigAsync(),                    
                Localization = await GetLocalizationConfigAsync(),
```


When I comment the line 89, all becomes well.

Step into `GetFeaturesConfigAsync()`,  exception occurs below.

`result.Values[featureDefinition.Name] = await FeatureChecker.GetOrNullAsync(featureDefinition.Name);`

**Comment by realLiangshiwei** [comment]
microservice sample has updated to latest. You can refer it

**Comment by GotzeWong** [comment]
I tried install package 
`<PackageReference Include="Volo.Abp.FeatureManagement.HttpApi.Client" Version="4.3.3" />`
 but not include package with lastest codes  		
`<ProjectReference Include="..\..\..\abp\modules\feature-management\src\Volo.Abp.FeatureManagement.HttpApi.Client\Volo.Abp.FeatureManagement.HttpApi.Client.csproj" />`.

And it's back to normal again. I will leave it later to figure out what happened here.

**Comment by GotzeWong** [comment]
Solved in #9439
